### PR TITLE
[StructuralMechanicsApplication] Fixes  #6839

### LINKED ISSUE #6839: StructuralMechanics errors after #6686
this is the error @loumalouomega (happens only in `FullDebug`)

~~~
======================================================================
ERROR: test_execution (structural_mechanics_test_factory.RigidFaceTestWithImposeRigidMovementProcess)
----------------------------------------------------------------------
Traceback (most recent call last):
  File "/home/philippb/software/Kratos_Master/install/applications/StructuralMechanicsApplication/tests/structural_mechanics_test_factory.py", line 63, in setUp
    self.test.Initialize()
  File "/home/philippb/software/Kratos_Master/install/KratosMultiphysics/StructuralMechanicsApplication/structural_mechanics_analysis.py", line 53, in Initialize
    super(StructuralMechanicsAnalysis, self).Initialize()
  File "/home/philippb/software/Kratos_Master/install/KratosMultiphysics/analysis_stage.py", line 87, in Initialize
    process.ExecuteInitialize()
  File "/home/philippb/software/Kratos_Master/install/KratosMultiphysics/StructuralMechanicsApplication/impose_rigid_movement_process.py", line 99, in ExecuteInitialize
    self.rigid_movement_process.ExecuteInitialize()
RuntimeError: Error: The component "DISPLACEMENT_X" is not registered!
Maybe you need to import the application where it is defined?
The following components of this type are registered:



in kratos/includes/kratos_components.h:100:static const TComponentType& KratosComponents<TComponentType>::Get(const string&) [with TComponentType = VariableComponent<VectorComponentAdaptor<array_1d<double, 3ul> > >; __cxx11::string = __cxx11::basic_string<char>]
   /home/philippb/software/Kratos_Master/applications/StructuralMechanicsApplication/custom_processes/impose_rigid_movement_process.cpp:163:virtual void ImposeRigidMovementProcess::ExecuteInitialize()


======================================================================
ERROR: test_execution (structural_mechanics_test_factory.RigidBlockTest)
----------------------------------------------------------------------
Traceback (most recent call last):
  File "/home/philippb/software/Kratos_Master/install/applications/StructuralMechanicsApplication/tests/structural_mechanics_test_factory.py", line 63, in setUp
    self.test.Initialize()
  File "/home/philippb/software/Kratos_Master/install/KratosMultiphysics/StructuralMechanicsApplication/structural_mechanics_analysis.py", line 53, in Initialize
    super(StructuralMechanicsAnalysis, self).Initialize()
  File "/home/philippb/software/Kratos_Master/install/KratosMultiphysics/analysis_stage.py", line 87, in Initialize
    process.ExecuteInitialize()
  File "/home/philippb/software/Kratos_Master/install/KratosMultiphysics/StructuralMechanicsApplication/impose_rigid_movement_process.py", line 99, in ExecuteInitialize
    self.rigid_movement_process.ExecuteInitialize()
RuntimeError: Error: The component "DISPLACEMENT_X" is not registered!
Maybe you need to import the application where it is defined?
The following components of this type are registered:



in kratos/includes/kratos_components.h:100:static const TComponentType& KratosComponents<TComponentType>::Get(const string&) [with TComponentType = VariableComponent<VectorComponentAdaptor<array_1d<double, 3ul> > >; __cxx11::string = __cxx11::basic_string<char>]
   /home/philippb/software/Kratos_Master/applications/StructuralMechanicsApplication/custom_processes/impose_rigid_movement_process.cpp:163:virtual void ImposeRigidMovementProcess::ExecuteInitialize()


======================================================================
ERROR: test_execution (structural_mechanics_test_factory.RigidEliminationTest)
----------------------------------------------------------------------
Traceback (most recent call last):
  File "/home/philippb/software/Kratos_Master/install/applications/StructuralMechanicsApplication/tests/structural_mechanics_test_factory.py", line 63, in setUp
    self.test.Initialize()
  File "/home/philippb/software/Kratos_Master/install/KratosMultiphysics/StructuralMechanicsApplication/structural_mechanics_analysis.py", line 53, in Initialize
    super(StructuralMechanicsAnalysis, self).Initialize()
  File "/home/philippb/software/Kratos_Master/install/KratosMultiphysics/analysis_stage.py", line 87, in Initialize
    process.ExecuteInitialize()
  File "/home/philippb/software/Kratos_Master/install/KratosMultiphysics/StructuralMechanicsApplication/impose_rigid_movement_process.py", line 99, in ExecuteInitialize
    self.rigid_movement_process.ExecuteInitialize()
RuntimeError: Error: The component "DISPLACEMENT_X" is not registered!
Maybe you need to import the application where it is defined?
The following components of this type are registered:



in kratos/includes/kratos_components.h:100:static const TComponentType& KratosComponents<TComponentType>::Get(const string&) [with TComponentType = VariableComponent<VectorComponentAdaptor<array_1d<double, 3ul> > >; __cxx11::string = __cxx11::basic_string<char>]
   /home/philippb/software/Kratos_Master/applications/StructuralMechanicsApplication/custom_processes/impose_rigid_movement_process.cpp:163:virtual void ImposeRigidMovementProcess::ExecuteInitialize()


======================================================================
ERROR: test_execution (structural_mechanics_test_factory.RigidSphereFailingExplicit)
----------------------------------------------------------------------
Traceback (most recent call last):
  File "/home/philippb/software/Kratos_Master/install/applications/StructuralMechanicsApplication/tests/structural_mechanics_test_factory.py", line 63, in setUp
    self.test.Initialize()
  File "/home/philippb/software/Kratos_Master/install/KratosMultiphysics/StructuralMechanicsApplication/structural_mechanics_analysis.py", line 53, in Initialize
    super(StructuralMechanicsAnalysis, self).Initialize()
  File "/home/philippb/software/Kratos_Master/install/KratosMultiphysics/analysis_stage.py", line 89, in Initialize
    self._GetSolver().Initialize()
  File "/home/philippb/software/Kratos_Master/install/KratosMultiphysics/StructuralMechanicsApplication/structural_mechanics_explicit_dynamic_solver.py", line 85, in Initialize
    super(ExplicitMechanicalSolver, self).Initialize()
  File "/home/philippb/software/Kratos_Master/install/KratosMultiphysics/StructuralMechanicsApplication/structural_mechanics_solver.py", line 212, in Initialize
    mechanical_solution_strategy.Initialize()
RuntimeError: Error: The component "DISPLACEMENT_X" is not registered!
Maybe you need to import the application where it is defined?
The following components of this type are registered:




in kratos/includes/kratos_components.h:100:static const TComponentType& KratosComponents<TComponentType>::Get(const string&) [with TComponentType = VariableComponent<VectorComponentAdaptor<array_1d<double, 3ul> > >; __cxx11::string = __cxx11::basic_string<char>]
   kratos/utilities/constraint_utilities.cpp:374:void ConstraintUtilities::PreComputeExplicitConstraintMassAndInertia(ModelPart&, __cxx11::string, __cxx11::string, __cxx11::string, __cxx11::string)
   /home/philippb/software/Kratos_Master/applications/StructuralMechanicsApplication/custom_strategies/custom_strategies/mechanical_explicit_strategy.hpp:270:void MechanicalExplicitStrategy<TSparseSpace, TDenseSpace, TLinearSolver>::Initialize() [with TSparseSpace = UblasSpace<double, boost::numeric::ublas::compressed_matrix<...>, boost::numeric::ublas::vector<double> >; TDenseSpace = UblasSpace<double, boost::numeric::ublas::matrix<double>, boost::numeric::ublas::vector<double> >; TLinearSolver = LinearSolver<UblasSpace<double, boost::numeric::ublas::compressed_matrix<...>, boost::numeric::ublas::vector<double> >, UblasSpace<double, boost::numeric::ublas::matrix<double>, boost::numeric::ublas::vector<double> > >]
~~~

### PR DESCRIPTION
**Pull request by loumalouomega**
Fixes  #6839

### CODE REVIEW

File: applications/StructuralMechanicsApplication/custom_processes/impose_rigid_movement_process.cpp [hunk 1]
```diff
@@ -63,53 +63,44 @@ void ImposeRigidMovementProcess::ExecuteInitialize()
 
     // Getting list of variables
     std::vector<const Variable<double>*> master_double_list_variables, slave_double_list_variables;
```
Comment (philbucher) [inline applications/StructuralMechanicsApplication/custom_processes/impose_rigid_movement_process.cpp]:
```suggestion
    std::vector<const Variable<double>*> master_variables, slave_variables;
```

**Comment by loumalouomega** [comment]
Can we live with the former name?

**Comment by philbucher** [comment]
I think now would be the time to clean it up but up to you
CI anyway fails

**Comment by loumalouomega** [comment]
OK, it was a risky move, I am doing something right now and I change it
without check the tests, sorry.

**Review by philbucher — APPROVED** [review]
thx

**Pull request merged**